src/core/state/currentSettings/connectedToHardhat.ts

```diff
@@ -22,4 +30,4 @@ export const connectedToHardhatStore = createStore<ConnectedToHardhatState>(
   },
 );
 
-export const useConnectedToHardhatStore = create(connectedToHardhatStore);
```

Comment:
oh the naming thing was just for the `persist` param, we do use the `Store` end on all of these